curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-recursion-by-building-a-decimal-to-binary-converter/645cc5925f158b5b33e2698f.md

```diff
@@ -9,9 +9,9 @@ dashedName: step-48
 
 When your code runs, the `a()` function is added to the call stack first.
 
-In your `callStack` array, add the following string: `"a(): returns 'freeCodeCamp ' + b()"`. This represents the function call and the code that will be executed.
+In your `callStack` array, add the following string: `'a(): returns "freeCodeCamp " + b()'`. This represents the function call and the code that will be executed.
 
-Note: Since the string you're adding includes double quotation marks, wrap it in single quotation marks (`'`) or backticks (`).
+Note: Since the string you're adding includes double quotation marks (`"`), wrap it in single quotation marks (`'`) or backticks (&#96;).
```

Comment:
Great, I didn't know how to go with that. The backtick may still give some issues when converting to translatable, as the parser there substitute the backtick with `<code>` tags, I don't know how it would behave with that, but we can check that later I guess. It's just something to keep in mind.